lib/deliver/runner.rb

```diff
@@ -19,7 +19,7 @@ def login
     def run
       verify_version if options[:app_version].to_s.length > 0
       upload_metadata
-      upload_binary if options[:ipa]
+      abort "Upload binary failed" unless upload_binary if options[:ipa]
```

Comment:
I don't like how we have `abort`, `unless` and `if` in one line. Could you refactor this to make it easier and clearer to read?